src/WeSay.ConfigTool/Tasks/TaskListPresentationModel.cs

```diff
@@ -57,6 +72,15 @@ private void OnProject_WritingSystemChanged(object sender, WeSayWordsProject.Str
 			}
 		}
 
+		private void OnProject_MeaningFieldChanged(object sender, WeSayWordsProject.StringPair pair)
+		{
+			foreach (var task in ICareThatMeaningFieldChangedTasks)
```

Comment:
I prefer only using `var` when the type is explicitly stated elsewhere in the statement. It makes it easier for others to review your code outside of an IDE.